national-resources/index.md

```diff
@@ -2,5 +2,6 @@
 title: National resources
 ---
 
+The national resources pages below provide a general overview of resources, tools, and data related to the management of infectious disease data in a given country. Given the recent COVID-19 pandemic, many of the resources featured on the pages are centered around COVID-19. However, the practices, regulations, and other resources can be repurposed for use with other pathogens.
```

Comment:
"Given the recent..." is this going to age? Maybe delete "Given the recent COVID-19 pandemic,"?

Reply:
Suggested change to account for this